source/manuals/security-overview-for-websites.html.md.erb

```diff
@@ -127,7 +126,9 @@ Enable relevant security headers to help your users' browsers understand what yo
 |Cross-Origin-Embedder-Policy|Sets the permission for how assets are loaded|
 |Cross-Origin-Opener-Policy|^|
 |Cross-Origin-Resource-Policy|^|
-|X-XSS-Protection|Enables older browsers' functionality to prevent XSS (cross-site scripting)|
+|Feature-Policy|*Partially deprecated* (see [OWASP guidance on Feature-Policy](https://owasp.org/www-project-secure-headers/#feature-policy)). Previously used to specify features the browser could access, primarily replaced with `Permissions-Policy`|
+|Expect-CT|*Deprecated* (see [OWASP guidance on Expect-CT](https://owasp.org/www-project-secure-headers/#expect-ct)). Previously used by browsers to check the certificate transparency logs, which helps determine the legitimacy of a HTTPS connection. It is no longer needed as a response header|
+|X-XSS-Protection|*Deprecated* (see [OWASP guidance on X-XSS-Protection](https://owasp.org/www-project-secure-headers/#x-xss-protection)). Previously used to mitigate XSS (cross-site scripting), this has been deprecated and should be set to `0` to disable the functionality in older browsers|
```

Comment:
Sorry for the late comment but I just want to double check what the deprecated part is telling me.

I believe this is saying, include the headers, acknowledging that they are deprecated so may be of limited value?

But I want to check it is not saying the opposite, we have deprecated our advice to include these headers and therefore you should no longer include them?